cms/src/test/resources/test-data/api-specifications/SwaggerCodeGenOpenApiSpecificationJsonToHtmlConverterTest/openapi-v3-spec-empty-res-sec.json

```diff
@@ -0,0 +1,4019 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+        "version": "v1.2.29-beta",
+        "title": "Personal Demographics Service (FHIR) API",
```

Comment:
I'm afraid this file still contains a lot of content that does nothing to illustrate the test but, instead, obscures the content that is relevant. The same applies to the corresponding HTML file.

For example the `info.description` field has a very large content but has zero impact on the test; no only its content is irrelevant, but even the presence of the field itself is redundant. All we need is a minimal, necessary content that surrounds the one that the test actually cares about.

Admittedly, we do use an almost complete PDS spec content in the `getHtmlForSpec_convertsProvidedOpenApiJsonToHtml` but this is a 'catch all', high level tests, written in the time when we had very little understanding of the specifications' content and PDS was our exemplar. Even that content is too big and full of redundancies and there is a tech debt ticket to clean it up.

For the test cases that are small and focused (like the one that uses the current file), the content should be in order of tens of lines, not five thousands as it's very hard to see what's relevant. File `exampleOnMediaTypeObjectNoneOnSchema.json` is a good example of a scale of small test data file that focuses only relevant data; naturally the current file needs to be slightly bigger but not by much.

Reply:
Fixed